src/ui-test/camelProjectCreateTest.ts

```diff
@@ -0,0 +1,95 @@
+        let driver: WebDriver;
+
+        before(async function() {
+            this.timeout(10000);
+            homedir = os.homedir() + path.sep + DIR;
+            driver = VSBrowser.instance.driver;
+            if (! fs.existsSync(homedir)) {
+                fs.mkdirSync(homedir);
+            }
+            await openCommandPrompt();
+            const quick = await QuickOpenBox.create();
+            await quick.setText(">Extest: Add Folder");
+            await quick.confirm();
+            let confirmedPrompt = await InputBox.create();
+            await confirmedPrompt.setText(homedir);
+            await confirmedPrompt.confirm();
+        });
+
+        CAMEL_MISSIONS_EXPECTED.forEach( async function(mission) {
+            it('Test creating Camel/Fuse project: ' + mission + ", runtime & version: " + RUNTIME_VERSION, async function () {
+                this.timeout(20000); 
+                const commandPrompt = await openCommandPrompt();
+                await commandPrompt.setText('>Project: ');
+                await commandPrompt.selectQuickPick(ProjectInitializer.PI_GENERAL.camel);
+                inputBox = await InputBox.create();
+                await inputBox.selectQuickPick(mission);
+                inputBox = await InputBox.create();
+                let index = await getIndexOfQuickPickItem(RUNTIME_VERSION, await inputBox.getQuickPicks());
+                await inputBox.selectQuickPick(index);
+                // settings default groupID
+                inputBox = await InputBox.create();
+                await inputBox.confirm();
+                // setting default artifactID
+                inputBox = await InputBox.create();
+                await inputBox.confirm();
+                // setting default version
+                inputBox = await InputBox.create();
+                await inputBox.confirm();
+                // select home dir
+                inputBox = await InputBox.create();
+                await inputBox.selectQuickPick(DIR);
+
+                // check the notification "Project saved to $homedir";
+                const notification = await driver.wait(() => { return notificationExists('Project saved to ' + homedir); }, 5000) as Notification;
+                await driver.actions().mouseMove(notification).perform();
+                await notification.findElement(By.className("codicon-close")).click();
+                // check explorer that it contains created project
+                const explorerView = await new ActivityBar().getViewControl("Explorer").openView();
+                const viewTab = await explorerView.getContent().getSection("Untitled (Workspace)");
+                let tree = await viewTab.getVisibleItems() as TreeItem[];
+                let items = tree.map( (item: TreeItem) => item.getLabel());
+                expect(items).to.contains.members(["pom.xml"]);
+            });
+        });
+
+        afterEach(async function () {
+            this.timeout(5000);
```

Comment:
Can you try to use a conditional wait?
Have you spotted something missing to use ones?